packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -68,6 +68,60 @@ const getAxisTypeFromField = (item?: Field): string => {
     }
 };
 
+type GetAxisTypeArg = {
+    validCartesianConfig: CartesianChart;
+    itemMap: Record<string, Field | TableCalculation>;
+    topAxisXId?: string;
+    bottomAxisXId?: string;
+    rightAxisYId?: string;
+    leftAxisYId?: string;
+};
+const getAxisType = ({
```

Comment:
beautiful

Reply:
Is this a new bug with datetimes ? 
![image](https://user-images.githubusercontent.com/1983672/182645225-b3934b9f-8609-42f4-84b4-b9271e45629f.png)

Reply:
Nice catch! We can support 'date' type on the y axis for horizontal bars. We only need to force 'category' when the default type is 'value'.

e.g of date in the y axis:
<img width="1243" alt="Screenshot 2022-08-03 at 16 27 18" src="https://user-images.githubusercontent.com/9117144/182647719-27b8dce1-49b1-41c0-b963-fdef18b4ff69.png">